winpr/include/winpr/sspi.h

```diff
@@ -598,19 +598,7 @@ typedef struct
 
 #define SEC_WINNT_AUTH_IDENTITY_ANSI 0x1
 #define SEC_WINNT_AUTH_IDENTITY_UNICODE 0x2
-
-typedef struct
-{
-	char* cache;
-	char* armorCache;
-	char* pkinitX509Anchors;
-	char* pkinitX509Identity;
-	BOOL withPac;
-	INT32 startTime;
-	INT32 renewLifeTime;
-	INT32 lifeTime;
-	BYTE certSha1[20];
-} SEC_WINPR_KERBEROS_SETTINGS;
+#define SEC_WINNT_AUTH_IDENTITY_EXTENDED 0x4
```

Comment:
I'd suggest to choose another value as it seems like it is already used in `SEC_WINNT_AUTH_IDENTITY_EXW.flags`for `SEC_WINNT_AUTH_IDENTITY_MARSHALLED`